### ISSUE TITLE
Is YOLOv8 considered a pre-train (transfer learning) algorithm?

### ISSUE THREAD

**Issue by e9l7a5f** [ultralytics/ultralytics#18514]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I did a face recognition, used my own dataset, and I have trained my model using YOLOv8.
Now I am confused by the term to be used to describe the method. Is YOLO considered a transfer learning method ?pre-train? or it is a fine-tune?
note: I didn't change anything during the training, not the learning rate, not even the batch size, not even the image size.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @e9l7a5f, thank you for your interest in Ultralytics 🚀! YOLO terminology can sometimes be a little confusing, but not to worry, we’re here to help!

If this is a custom training ❓ question, please provide additional details about your training process, such as any specific configurations, dataset summaries, or logs. It’s great to know that you’ve used your own dataset and default settings, but further details will help ensure a clear and helpful discussion.

If your question involves any nuances related to transfer learning, pre-training, or fine-tuning, and you're looking for a formal definition specific to your case, an Ultralytics engineer will follow up with further guidance soon.

While waiting, feel free to check out our documentation that clarifies these concepts, and join the vibrant Ultralytics community if you'd like to connect with others discussing similar topics. You can also upgrade to the latest `ultralytics` release to ensure you have access to the newest features and best performance.

Commands for upgrading:
```bash
pip install -U ultralytics
```

We recommend running YOLO in a verified environment for the best experience. Whether you're using Jupyter notebooks, cloud services, or Docker setups, ensure that all dependencies, including Python, PyTorch, CUDA, and CUDNN, are up-to-date.

If this is unrelated to custom training or requires deeper inspection, don’t worry — an Ultralytics engineer will assist you soon. Stay tuned and keep exploring 🚀!

**Comment by glenn-jocher** [comment]
@e9l7a5f yes, YOLOv8 can be considered part of a transfer learning workflow when you train it on your own dataset, as it leverages pre-trained weights (e.g., COCO) to adapt to new tasks. The process you described aligns with fine-tuning, a specific application of transfer learning, where a pre-trained model is refined using your dataset. You can learn more about fine-tuning [here](https://www.ultralytics.com/glossary/fine-tuning).

**Comment by visionNoob** [comment]
[Docs](https://docs.ultralytics.com/modes/train/#resuming-interrupted-trainings) also helpful to train your own dataset. 

In `Training Setting`, many options for fine-tuning (`pretrained`, `freeze`, etc) are explaned.

**Comment by glenn-jocher** [comment]
@visionNoob thank you for pointing that out! Indeed, the [Training Settings docs](https://docs.ultralytics.com/modes/train/#resuming-interrupted-trainings) provide valuable insights for fine-tuning with options like `pretrained` and `freeze` to optimize training on custom datasets.
